### PR TITLE
Jet Punch animation

### PR DESCRIPTION
**Pull request by PCG06**
## Description
Adds a simple animation for Jet Punch.

## Images
jetpunch1

## **Discord contact info**
pcg06

### CODE REVIEW

**Comment by PCG06** [comment]
Here's another version of the animation I made awhile before this

jetpunch2

**Comment by Jaizu** [comment]
Mix of tabs and spaces

**Comment by PCG06** [comment]
> Mix of tabs and spaces

Done

**Comment by ghoulslash** [comment]
The second version seems more true to the SV animation so I would probably vote for that one. Might look better using the waterfall hits instead of the dive sprite?

**Comment by PCG06** [comment]
> The second version seems more true to the SV animation so I would probably vote for that one. Might look better using the waterfall hits instead of the dive sprite?

Thank you! I'll try that

**Comment by PCG06** [comment]
Does this look good enough?

jetpunch2

**Comment by Bassoonian** [comment]
I like it a lot!

**Comment by PCG06** [comment]
Can the PR be accepted or does the animation need some more improvements?

File: data/battle_anim_scripts.s [hunk 1]
```diff
@@ -17187,12 +17187,55 @@ Move_UPPER_HAND::
...
+	loadspritegfx ANIM_TAG_ICE_CRYSTALS
+	loadspritegfx ANIM_TAG_HANDS_AND_FEET
+	loadspritegfx ANIM_TAG_IMPACT
+	loadspritegfx ANIM_TAG_SPLASH
+	loadspritegfx ANIM_TAG_WATER_IMPACT
+	loadspritegfx ANIM_TAG_SMALL_BUBBLES
+	monbg ANIM_DEF_PARTNER
+	setalpha 12, 8
+	playsewithpan SE_M_DIVE, SOUND_PAN_TARGET
+	createvisualtask AnimTask_AttackerStretchAndDisappear, 2
```
Comment (AsparagusEduardo) [inline data/battle_anim_scripts.s:17200]: Trailing whitespace

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]
Did you test on doubles?

**Comment by PCG06** [comment]
> Did you test on doubles?

Sorry for the delay, I was super busy with school. But I will send a gif within a few hours. Sorry :)

**Comment by PCG06** [comment]
> Did you test on doubles?

This is weird 🤔 
jetpunch

**Comment by PCG06** [comment]
The issue came from `monbg ANIM_TARGET`. Clearing it didn't solve it, but removing it did.

jetpunch

**Review by ghoulslash — APPROVED** [review]

**Review by PCG06 — COMMENTED** [review]
I've done the changes. Can this PR be merged or are there any improvements suggested?

File: data/battle_anim_scripts.s (hunk 1 above)
Comment (AlexOn1ine) [reply data/battle_anim_scripts.s:17200]: Edu, do you have any follow up? :eyes:

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**